prereise/gather/demanddata/bldg_electrification/load_projection.py

```diff
@@ -0,0 +1,674 @@
+                self.stats[f"ff2hp_{clas}"] = (
+                    1
+                    - self.stats[f"frac_ff_{clas}_heat"]
+                    / other.stats[f"frac_ff_{clas}_heat"]
+                )
+            elif not np.isnan(self.stats[f"ff2hp_{clas}"]):
+                self.stats[f"frac_hp_{clas}_heat"] = (
+                    other.stats[f"frac_hp_{clas}_heat"]
+                    + other.stats[f"frac_resist_{clas}_heat"]
+                    * self.stats[f"resist2hp_{clas}"]
+                    + other.stats[f"frac_ff_{clas}_heat"] * self.stats[f"ff2hp_{clas}"]
+                )
+                self.stats[f"frac_resist_{clas}_heat"] = other.stats[
+                    f"frac_resist_{clas}_heat"
+                ] * (1 - self.stats[f"resist2hp_{clas}"])
+                self.stats[f"frac_ff_{clas}_heat"] = other.stats[
+                    f"frac_ff_{clas}_heat"
+                ] * (1 - self.stats[f"ff2hp_{clas}"])
+            else:
+                self.stats[f"frac_hp_{clas}_heat"] = other.stats[f"frac_hp_{clas}_heat"]
+                self.stats[f"frac_resist_{clas}_heat"] = other.stats[
+                    f"frac_resist_{clas}_heat"
+                ]
+                self.stats[f"frac_ff_{clas}_heat"] = other.stats[f"frac_ff_{clas}_heat"]
+                self.stats[f"ff2hp_{clas}"] = 0
+                self.stats[f"resist2hp_{clas}"] = 0
+
+            if np.isnan(self.stats[f"frac_elec_{clas}_cool"]):
+                self.stats[f"frac_elec_{clas}_cool"] = other.stats[
+                    f"frac_elec_{clas}_cool"
+                ]
+            if np.isnan(self.stats[f"frac_ff_dhw_{clas}"]):
+                self.stats[f"frac_ff_dhw_{clas}"] = other.stats[f"frac_ff_dhw_{clas}"]
+            cook_other = "cook" if clas == "com" else "other"
+            if np.isnan(self.stats[f"frac_ff_{cook_other}_{clas}"]):
+                self.stats[f"frac_ff_{cook_other}_{clas}"] = other.stats[
+                    f"frac_ff_{cook_other}_{clas}"
+                ]
+
+        self.hp_heat_area_m2 = (
+            self.stats["frac_hp_res_heat"] * self.floor_area_m2["res"]
+            + self.stats["frac_hp_com_heat"] * self.floor_area_m2["com"]
+        )
+
+        self.hp_heat_frac = self.hp_heat_area_m2 / (
+            self.floor_area_m2["res"] + self.floor_area_m2["com"]
+        )
+        self.resist_heat_area_m2 = (
+            self.stats["frac_resist_res_heat"] * self.floor_area_m2["res"]
+            + self.stats["frac_resist_com_heat"] * self.floor_area_m2["com"]
+        )
+        self.resist_heat_frac = self.resist_heat_area_m2 / (
+            self.floor_area_m2["res"] + self.floor_area_m2["com"]
+        )
+        self.elec_cool_m2 = (
+            self.stats["frac_elec_res_cool"] * self.floor_area_m2["res"]
+            + self.stats["frac_elec_com_cool"] * self.floor_area_m2["com"]
+        )
+        return
```

Comment:
I think we don't need this `return` here.

Reply:
done